### PR TITLE
Made tag for new job listing

### PR DESCRIPTION
**Pull request by solveigua**
:) takk for meg klem fra gruppeansvarlig Media ❤️

### CODE REVIEW

File: app/routes/joblistings/components/JoblistingList.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import { Image } from 'app/components/Image';
 import Time from 'app/components/Time';
 import { Flex } from 'app/components/Layout';
 import { Year, jobType, Workplaces } from './Items';
+import Tag from 'app/components/Tags/Tag';
+import moment from 'moment';
```
Comment (SmithPeder) [inline app/routes/joblistings/components/JoblistingList.js:10]: I'm quite sure we don't have `moment`, but rather `moment-timezone` 😄 
This should in theory don't work 😕 

`import moment from 'moment-timezone';`

Comment (solveigua) [reply app/routes/joblistings/components/JoblistingList.js]: fixed it 😎😎😎😎😎

File: cypress/support/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import './commands';
 
 Cypress.on('uncaught:exception', (err, runnable, promise) => {
   console.log(err.payload.response.status);
-  if (promise && err.payload && err.payload.response.status == 404) {
+  if (promise && err.payload && err.payload.response.status === 404) {
```
Comment (odinuge) [inline cypress/support/index.js:5]: :trollface:

File: app/routes/joblistings/components/JoblistingList.js [hunk 3]
```diff
@@ -30,7 +32,12 @@ function JoblistingItem({ joblisting }: JobListingItemProps) {
         <div>
           <Link to={`/joblistings/${joblisting.id}/`}>
             <Flex>
-              <h3 className={styles.joblistingItemTitle}>{joblisting.title}</h3>
+              <h3 className={styles.joblistingItemTitle}>
+                {moment(joblisting.createdAt).isAfter(
+                  moment().subtract(3, 'days')
+                ) && <Tag tag="NY" color="green"></Tag>}
```
Comment (odinuge) [inline app/routes/joblistings/components/JoblistingList.js:38]: Would be nice with a screenshot tho.

File: app/components/Tags/Tag.css [hunk 4]
```diff
@@ -58,6 +58,15 @@ html[data-theme='dark'] .cyan {
   border-color: #fde3cf;
 }
 
+.green {
+  background: greenyellow;
+  border-color: aquamarine;
+}
+
+html[data-theme='dark'] .green {
+  color: black;
```
Comment (odinuge) [inline app/components/Tags/Tag.css:67]: class `green` -> color `black` :tm:

**Review by odinuge — APPROVED** [review]
nits

otherwise good

/lgtm

**Pull request merged**